userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -337,3 +337,35 @@ Both `head.html` and `scripts.html` are then used to build Docsy's [base page la
 </html>
 ```
 
+## Adding custom class to the body element
+
+By default, Docsy adds the `td-{{ .Kind }}` class, where the kind is the kind of the page, like section, blog, and so on. For example:
+
+```html
+<body class="td-section">
+```
+
+Sometimes it's useful to assign custom classes to a page, or to an entire section, for example, to apply custom styling. To do so, add the `body_class` parameter to the frontmatter of your page. The value of the parameter will be then added to the class attribute of your page's body element.
```

Comment:
Please line-wrap all long paragraphs in this delta.

Also:

- "will be then" --> "will then be"

Reply:
Ping for the above. Also pls resolved conflicts.